src/main/java/racingcar/domain/util/RandomNumber.java

```diff
@@ -0,0 +1,21 @@
+package racingcar.domain.util;
+
+import racingcar.exception.MoveConditionOutOfBoundException;
+
+import java.util.Random;
+
+public class RandomNumber {
+    private static final int MINIMUM = 0;
+
+    private static Random random = new Random();
+
+    public static int generate(int bound) {
+        int randomNumber = random.nextInt(bound + 1);
+
+        if (!(MINIMUM <= randomNumber && randomNumber <= bound)) {
```

Comment:
nextInt에서 이미 양수값을 반환하기 때문에 불필요한 비교연산입니다.
```suggestion
        if (randomNumber > bound)
```
이렇게 해도 결과는 같지 않을까요?

Reply:
네 맞습니다! 5bc0bb2 에 반영되었습니다